AnnoyancesFilter/sections/popups.txt

```diff
@@ -109,6 +109,7 @@ quirkbooks.com#$#.newsletterPrompt { display: none !important; }
 quirkbooks.com#$?#html { overflow: visible !important; touch-action: auto !important; }
 bing.com##.ev_talkbox_wrapper
 gizbot.com###containerPop
+weather.com##main > div[id^="Taboola-main-"]
```

Comment:
Do you still have it? I can't reproduce.

Reply:
Not anymore, but it is still in the DOM.

Reply:
Yes, but it has another content. Also it seems that `Easylist` fixed it.